### PR TITLE
New package: DuckStudio.FufuTools version 1.3.1-1

### LINKED ISSUE #156080: [Package Request]: Fufu Tools v1.3.1-1
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url:`https://github.com/DuckDuckStudio/Fufu_Tools/releases/`
Publisher:`鸭鸭「カモ」` (or `Duck Studio`)
Package Name:`芙芙工具箱` (or `Fufu Tools`)
Description:`一个开源的实用工具箱`
Package Version:`v1.3.1-1`
Installer URL:`https://github.com/DuckDuckStudio/Fufu_Tools/releases/download/v1.3.1-1/Fufu_Tools_Setup.v1.3.1-1.exe`

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Related #156080 but didn't solve it.

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

> [!IMPORTANT]
>
> Since @DuckDuckStudio create the related issue, and he/she is the publisher & author of the package. This pull request should also ask for approval from him/she.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/156134)

### CODE REVIEW

File: manifests/d/DuckDuckStudio/Fufu_Tools/1.3.1-1/DuckDuckStudio.Fufu_Tools.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+PackageIdentifier: DuckDuckStudio.Fufu_Tools
+PackageVersion: 1.3.1-1
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2024-06-01
```
Comment (DuckDuckStudio) [inline manifests/d/DuckDuckStudio/Fufu_Tools/1.3.1-1/DuckDuckStudio.Fufu_Tools.installer.yaml:12]:
```suggestion
ReleaseDate: 2024-05-29
```
这个应该是指发布日期,,,吧？

**Comment by DuckDuckStudio** [comment]
在旧PR中有提到静默安装参数

File: manifests/d/DuckDuckStudio/Fufu_Tools/1.3.1-1/DuckDuckStudio.Fufu_Tools.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: DuckDuckStudio.Fufu_Tools
+PackageVersion: 1.3.1-1
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
```
Comment (Dragon1573) [inline manifests/d/DuckDuckStudio/Fufu_Tools/1.3.1-1/DuckDuckStudio.Fufu_Tools.installer.yaml:10]:
```suggestion
- silentWithProgress
InstallerSwitches:
  Silent: /verysilent /suppressmsgboxes
  SilentWithProgress: /verysilent /suppressmsgboxes
```

**Comment by Dragon1573** [comment]
> Dragon1573,
> 
> The check-in policies require a moderator to approve PRs from the community. Our [moderators](https://github.com/microsoft/winget-pkgs/issues/15674#issue-908889039) are community volunteers, please be patient and allow them sufficient time to review your submission.

Hooray! @DuckDuckStudio 🎉 It seems your main application was built for `x64` only, but you packaged them into a `x86` installer wizard. So _Komac_ detected the architecture of wizard and also mark the whole package as `x86` instead of `x64`.

-----

好耶！看上去你的主程序是仅针对 `x64` 架构进行构建输出的，但你用打包工具把它制作成了一个 `x86` 架构的安装向导文件。 _Komac_ 检测到了安装向导的架构，于是也把你的主程序标记为 `x86` 架构了。

File: manifests/d/DuckDuckStudio/Fufu_Tools/1.3.1-1/DuckDuckStudio.Fufu_Tools.locale.zh-CN.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: DuckDuckStudio.Fufu_Tools
+PackageVersion: 1.3.1-1
+PackageLocale: zh-CN
+Publisher: 鸭鸭工作室
+PublisherUrl: https://github.com/DuckDuckStudio
```
Comment (DuckDuckStudio) [inline manifests/d/DuckDuckStudio/Fufu_Tools/1.3.1-1/DuckDuckStudio.Fufu_Tools.locale.zh-CN.yaml:8]:
```suggestion
PublisherUrl: https://duckduckstudio.github.io/yazicbs.github.io/
```

File: manifests/d/DuckDuckStudio/Fufu_Tools/1.3.1-1/DuckDuckStudio.Fufu_Tools.locale.zh-CN.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+Author: 鸭鸭「カモ」
+PackageName: 芙芙工具箱
+PackageUrl: https://github.com/DuckDuckStudio/Fufu_Tools
```
Comment (DuckDuckStudio) [inline manifests/d/DuckDuckStudio/Fufu_Tools/1.3.1-1/DuckDuckStudio.Fufu_Tools.locale.zh-CN.yaml:11]:
```suggestion
PackageUrl: https://duckduckstudio.github.io/yazicbs.github.io/Tools/Fufu_Tools/
```

**Review by DuckDuckStudio — APPROVED** [review]

**Comment by DuckDuckStudio** [comment]
And I have a request, could you please put this program in `DuckStudio` folder instead of `DuckDuckStudio` folder?
If possible, please also change the ID to `DuckStudio.FufuTools`.
Thank you.

---

如果可以，请将`DuckDuckStudio` 改为 `DuckStudio`，并修改ID。
感谢。

**Review by DuckDuckStudio — APPROVED** [review]

**Comment by DuckDuckStudio** [comment]
Why it failing again....

**Comment by Dragon1573** [comment]
@stephengillie - Could you please manually trigger the _Azure Pipelines_ again? 🙏🏼 The manifest can pass the `winget validate`, and able to install in `amd64` Windows Sandbox instance.

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978334964
> 2024-06-02 01:32:10.146 [CLI ] ShellExecute installer failed: 5
>                           An error occurred while trying to read the source file:

(Automated response - build 882.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> 2024-06-03 16:43:46.714 [FAIL] Installer failed security check. Url: https://github.com/DuckDuckStudio/Fufu_Tools/releases/download/v1.3.1-1/Fufu_Tools_Setup.v1.3.1-1.exe Result: 0x80004005

(Automated response - build 882.)

**Comment by Dragon1573** [comment]
@DuckDuckStudio - Seems there're security risks of the package and it's detected by _Manual Validation_ process.

Really wierd ... 😕

**Comment by Dragon1573** [comment]
> [!NOTE]
>
> The related issue has been resolved by another pull request. This PR can be closed as rejected.

**Pull request closed without merging**